porch/pkg/oci/oci.go

```diff
@@ -379,9 +381,11 @@ func (p *ociPackageRevision) Key() repository.PackageRevisionKey {
 	}
 }
 
-func (p *ociPackageRevision) GetPackageRevision() *v1alpha1.PackageRevision {
+func (p *ociPackageRevision) GetPackageRevision(ctx context.Context) *v1alpha1.PackageRevision {
 	key := p.Key()
 
+	kf, _ := p.GetKptfile(ctx)
```

Comment:
Nit: we shouldn't drop errors without checking them

Reply:
Yeah, I meant to clean this up but forgot about it. Fixed now.